collections/_sdk/effects/note.md

```diff
@@ -448,6 +448,35 @@ class Protocol(BaseHandler):
         return appointment_effect.cancel()
 ```
 
+## Managing Appointment Labels
+
+Canvas supports adding up to 3 labels per appointment for categorization and workflow automation. Labels can be managed programmatically using the appointment label effects.
+
+For detailed documentation on appointment label management, see [Appointment Label Effects](/sdk/effect-appointment-labels/).
+
+### Quick Example
+
+```python?partial=true
+from canvas_sdk.effects.note.appointment import AddAppointmentLabel, RemoveAppointmentLabel
+from canvas_sdk.handlers.base import BaseHandler
+
+class MyProtocol(BaseHandler):
```

Comment:
Call it `MyHandler` instead and provide a `RESPONDS_TO` for the class to make it a functional handler.